src/databases/WData/WDataCommonPluginInfo.C

```diff
@@ -0,0 +1,109 @@
+/*****************************************************************************
```

Comment:
I am not sure where lines 1...36 came from here and in a few other source files in this PR? Was it cut-n-paste from another source file? Or, did our `xml2plugin` tools wind up producing that?

In any case, please cut that down to the same 3-line copyright used elsewhere as in for example (totally random selection of a database plugin source file), https://github.com/visit-dav/visit/blob/f060e01861a317ce327623b85039251933b3d26b/src/databases/Image/avtImageFileFormat.C#L1-L3

I think it is perfectly fine to also add your own copyright statement if you would like, after ours here too and I would encourage you to do that if you are so inclined.

Reply:
The lines (1-36) are what **xml2** tool produced. I might use the old version by mistake (we have a visit 2.13 module on the local server) and that is the reason for that. Also, I used **xml2cmake** if that makes any difference. 
Included our licence to wdata.c/.h files